csrc/python_frontend/fusion_definition.h

```diff
@@ -182,6 +183,15 @@ class NVF_API FusionDefinition : public FusionState {
   //! Gets a Record State object
   NVF_API State recordingState(size_t index) const;
 
+  //! Experimental API we'll redo this later!
+  void setMultiDevice() {
+#ifdef NVFUSER_DISTRIBUTED
```

Comment:
We can use the `distributedEnabled()` helper function to clean things up a bit.
https://github.com/NVIDIA/Fuser/blob/main/csrc/multidevice/utils.h#L18

Reply:
Awesome. Thanks for that.